.rubocop_todo.yml

```diff
@@ -108,6 +108,20 @@ RSpec/DescribeClass:
 RSpec/ExampleLength:
   Max: 148
 
+# Offense count: 8
+# Configuration parameters: Include, CustomTransform, IgnoreMethods, SpecSuffixOnly.
+# Include: **/*_spec*rb*, **/spec/**/*
+RSpec/FilePath:
```

Comment:
https://docs.rubocop.org/rubocop-rspec/cops_rspec.html#rspecfilepath mentions it's deprecated. Perhaps we should instead configure the cop to be `SpecSuffixOnly` (which is a good check).

Reply:
Hi @ekohl.  I believe all rubocop checks are still available after excluding the ``Rspec/FilePath`` cop.  The new ``Rspec/SpecFilePathSuffix`` should cover things now so resolving this.  Let me know if you have any more comments though.